src/components/redaction/redaction-tags-tab.js

```diff
@@ -129,8 +129,8 @@ function RedactionTagsTab({ documents, mediaLibraryItems }) {
     setDisplayedTableRows(sortedRows);
   }, [allTableRows, filterText, currentTableSorting, tableSorters]);
 
-  const handleTableChange = ({ current, pageSize }) => {
-    setCurrentPagination([current, pageSize].join());
+  const handleTableChange = newPagination => {
+    setCurrentTablePagination(oldPagination => ({ ...oldPagination, ...newPagination }));
```

Comment:
We talked about persisting paging data in the URL for search only, probably for redaction it is not as necessary, but I streamlined the paging here too, for consistency.

Reply:
I'm fond of streamlining, so absolutely no objections :rofl:

Reply:
❤️